types/react-redux/index.d.ts

```diff
@@ -19,6 +19,10 @@ interface ComponentDecorator<TStateProps, TDispatchProps, TOwnProps> {
     (component: Component<TStateProps & TDispatchProps & TOwnProps>): ComponentClass<TOwnProps>;
 }
 
+interface MergedComponentDecorator<TOwnProps, TMergedProps> {
```

Comment:
I don't understand.
There isn't necessarily any relationship between `TMergedProps` and anything else, so I don't see how they'd be combined.